src/components/Trends/Explore/TrendsExploreSearch.js

```diff
@@ -5,7 +5,7 @@ import styles from './TrendsExploreSearch.module.scss'
 
 const TrendsExploreSearch = ({ topic }) => (
   <div className={styles.wrapper}>
-    <TrendsForm className={styles.search} defaultTopic={topic} />
+    <TrendsForm inputClassName={styles.search} defaultTopic={topic} />
```

Comment:
everything is ok. But I like React Material UI pattern for internal classNames.

they passed a prop classes = { input, ... }
![image](https://user-images.githubusercontent.com/522287/55806454-f02e7600-5ae8-11e9-9a17-6d5cfe1aa644.png)

What do you think? @DmitriVanGuard